pyuvsim/source.py

```diff
@@ -202,7 +315,8 @@ def update_positions(self, time, telescope_location):
             return
 
         skycoord_use = SkyCoord(self.ra, self.dec, frame='icrs')
-        source_altaz = skycoord_use.transform_to(AltAz(obstime=time, location=telescope_location))
+        source_altaz = skycoord_use.transform_to(AltAz(obstime=time,
+                                                       location=telescope_location))
```

Comment:
Below here, there's a transformation to cirs and then to tee_ra which is used to calculate the source hour angles for all sources, which was used by coherency rotation. It looks like that functionality is not used anymore, so we can probably get rid of it and avoid doing an unnecessary transform.

Reply:
yeah, we don't need hour angle. I'll take that out.